WordPress/Classes/Services/AccountService.h

```diff
@@ -84,16 +85,15 @@ extern NSNotificationName const WPAccountEmailAndDefaultBlogUpdatedNotification;
 ///-----------------------
 
 /**
- A convenience method for creating an Account just from an auth token.
- A temporary username is assigned and is expected to be updated as soon as
- the account is synced from the server.
+ A convenience method for creating an Account from an auth token and remote user.
 
  This method makes a pass through call to `createOrUpdateAccountWithUsername:authToken:`, see it for details.
 
+ @param remoteUser a RemoteUser instance.
  @param authToken the OAuth2 token returned by signIntoWordPressDotComWithUsername:authToken:
  @return a WordPress.com `WPAccount` object
  */
-- (WPAccount *)createOrUpdateAccountWithAuthToken:(NSString *)authToken;
+- (WPAccount *)createOrUpdateAccountWithUserDetails:(RemoteUser *)remoteUser authToken:(NSString *)authToken;
```

Comment:
Since this is only called from `AccountService.m` now, I think we can revert this bit on the `.h` file